sdk/sdk-api/src/main/java/mil/emp3/api/interfaces/core/storage/IMapStatus.java

```diff
@@ -37,7 +37,7 @@
     boolean isEditing();
 
```

Comment:
We need to update IMap where draw and edit API is defined to explain what the difference is between Draw and Edit,

1. You can Draw a new feature
2. You can put an existing feature in Draw mode.
3. How and when control points are added to the feature depending on the mode Draw or Edit?